### PR TITLE
Plannig Task List - Add Gantt tab

### PR DESCRIPTION
**Pull request by groshenkovamarina**

### CODE REVIEW

File: packages/angular/src/app/pages/planning-task-list/task-list-gantt/task-list-gantt.component.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<dx-gantt *ngIf="!isLoading" taskListWidth="500" scaleType="weeks" height="700">
```
Comment (dxvladislavvolkov) [inline packages/angular/src/app/pages/planning-task-list/task-list-gantt/task-list-gantt.component.html:1]: For property numbers use square brackets( [height]="700" )

File: packages/angular/src/app/pages/planning-task-list/task-list-gantt/task-list-gantt.component.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import {
+  Component, NgModule, Input, SimpleChanges, OnChanges,
+} from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { HttpClientModule } from '@angular/common/http';
+import { Subscription } from 'rxjs';
+import {
+  DxGanttModule,
+  DxLoadPanelModule,
+} from 'devextreme-angular';
```
Comment (dxvladislavvolkov) [inline packages/angular/src/app/pages/planning-task-list/task-list-gantt/task-list-gantt.component.ts:10]: Use long pah to module to improve tree shaking (devextreme-angular/ui/gantt)

File: packages/angular/src/app/pages/planning-task-list/task-list-gantt/task-list-gantt.component.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  selector: 'task-list-gantt',
+  templateUrl: './task-list-gantt.component.html',
+  styleUrls: ['./task-list-gantt.component.scss'],
+})
+export class TaskListGanttComponent implements OnChanges {
+  @Input() dataSource: Task[];
+
+  isLoading = true;
+
+  dataSubscription: Subscription = new Subscription();
```
Comment (dxvladislavvolkov) [inline packages/angular/src/app/pages/planning-task-list/task-list-gantt/task-list-gantt.component.ts:23]: Why do you init dataSubscription? Looks like you don't use it in the component

File: packages/angular/src/app/pages/planning-task-list/task-list-gantt/task-list-gantt.component.html (hunk 1 above)
Comment (groshenkovamarina) [reply packages/angular/src/app/pages/planning-task-list/task-list-gantt/task-list-gantt.component.html]: Fixed

File: packages/angular/src/app/pages/planning-task-list/task-list-gantt/task-list-gantt.component.ts (hunk 2 above)
Comment (groshenkovamarina) [reply packages/angular/src/app/pages/planning-task-list/task-list-gantt/task-list-gantt.component.ts]: Fixed

File: packages/angular/src/app/pages/planning-task-list/task-list-gantt/task-list-gantt.component.ts (hunk 3 above)
Comment (groshenkovamarina) [reply packages/angular/src/app/pages/planning-task-list/task-list-gantt/task-list-gantt.component.ts]: Fixed

**Review by dxvladislavvolkov — APPROVED** [review]

**Pull request merged**